playwright/tests/common/autoSync.spec.ts

```diff
@@ -44,7 +44,11 @@ test("I can see the auto-sync succeed when making a change", async ({
   await expect(pageTypesBuilderPage.menu.autoSyncSynced).toBeVisible();
 });
 
```

Comment:
⚠️ #coverage

I would quickly update the test when adding a field not in a group + in a group (both case) with the check of the success toaster, as it's a product requirement so we ensure we don't miss it one day.

Reply:
Done!